docs/questions/native-editor/filter-widgets.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Filter and parameter widgets for native code
+---
+
+# Filter and parameter widgets for native code
+
+When you add a [SQL variable or parameter](./sql-parameters.md) to your native/SQL query, Metabase will add a widget to the top of the query that people can use to set the variable's value.
+
+## How to create different types of filter and parameter widgets
+
+The kind of filter widget that Metabase displays when you create a field filter widget depends on a setting for that field in Metabase called **Filtering on this field**. Admins can set this field option to:
```

Comment:
The field filter part here is important but even I missed it while reading. I think there should be two sections - for field filter and for basic variables